spec/rspec/rails/example/view_example_group_spec.rb

```diff
@@ -248,19 +248,17 @@ def _view; end # Stub method
       end
 
       it 'is accessible to hooks', pending: pending_only_on_ruby_22_rails_52 do
-        with_isolated_config do
-          run_count = 0
-          RSpec.configuration.before(:each, type: :view) do
-            allow(view).to receive(:render) { :value }
-            run_count += 1
-          end
-          group = RSpec::Core::ExampleGroup.describe 'a view', type: :view do
-            specify { expect(true).to eq true }
-          end
-          group.run(failure_reporter)
-          expect(failure_reporter.exceptions).to eq []
-          expect(run_count).to eq 1
+        group = RSpec::Core::ExampleGroup.describe 'a view', type: :view do
```

Comment:
Can you restore this, it was a regression check specifically for configuration hooks added (7 years ago!!) in #833

Reply:
Aha, got it 👍

Reply:
Happy for the doc string / or a comment to be changed to mention this though, as I had to check to find it :joy: